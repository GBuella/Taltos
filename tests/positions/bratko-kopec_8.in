force
nopost
setboard 4b3/p3kp2/6p1/3pP2p/2pP1P2/4K1P1/P3N2P/8 w - -
nps 100000
st 50
search_sync
hash_value_min 1
