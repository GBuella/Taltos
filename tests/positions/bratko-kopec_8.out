1. f5
ok
